.githooks/pre-commit

```diff
@@ -734,4 +734,15 @@ if [ "$FILES_FIXED" = "1" ]; then
 else
     echo_success "All pre-commit checks passed."
 fi
+
+# Suggestion: Run full validation before creating PR
```

Comment:
@rjmurillo-bot A suggestion nobody will follow. If you need the AI agents to run it, then you need to make the message more like an actioanble prompt for them.